data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleCrac.java

```diff
@@ -20,12 +19,22 @@
  */
 public class SimpleCrac extends AbstractIdentifiable implements Crac {
     private List<Cnec> cnecs;
+    private List<Contingency> contingencies;
```

Comment:
Why did you change the format?

Reply:
It enables to create separately contingencies and cnecs